css/styles.css

```diff
@@ -65,3 +80,30 @@ button {
 .column-full.button {
   text-align: end;
 }
+
+ul {
+  padding: 0;
+  display: inline;
+}
+
+li {
+  display: inline;
+}
+
+.row.entry-block {
+  margin-bottom: 5%;
```

Comment:
Instead of a `%`, try to stick with an absolute value. I recommend small box model adjustments be written as `rem`.